src/it/resources/com/sun/checkstyle/test/chapter4indentation/rule41linelength/InputLineLength.java

```diff
@@ -0,0 +1,90 @@
+package com.sun.checkstyle.test.chapter4indentation.rule41linelength;
+
+public class InputLineLength
+{
+    int i;
+    int j;
+    int i1;
+    int j1;
+    // A very, very long line that is OK because it matches the regexp "^.*is OK.*regexp.*$" // warn
```

Comment:
This comment does not make sense in this context; if this line is ok, then why do we get a violation?